test/specs/modules/Progress/Progress-test.js

```diff
@@ -91,22 +91,22 @@ describe('Progress', () => {
   describe('data-percent', () => {
     it('adds prop by default', () => {
       shallow(<Progress />)
-        .should.have.prop('data-percent', undefined)
+        .should.have.data('percent')
```

Comment:
`chai-enzyme` updated `enzyme`, now `data-percent` equals `NaN`. I simplified this assert.